benchmarks/tpch/run.py

```
@@ -0,0 +1,355 @@
+
+
+ENGINE_DIALECT = "postgres"
+
+
+def arrow_to_rows(table):
+    """Convert a pyarrow Table into a list of positional row tuples."""
+    columns = []
+    for index in range(table.num_columns):
+        columns.append(table.column(index).to_pylist())
+    rows = []
+    for row in zip(*columns):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)